packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -15,7 +15,7 @@ import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable
  *         and L2. Messages sent directly to the OptimismPortal have no form of replayability.
  *         Users are encouraged to use the L1CrossDomainMessenger for a higher-level interface.
  */
-contract OptimismPortal is ResourceMetering, Initializable {
+contract OptimismPortal is Initializable, ResourceMetering {
```

Comment:
I realized that the Portal's `_initializing`/`_initialized` values were not in the first slot as they are in our other upgradable contracts. I thought that was worth preserving, and it meant I didn't have to do that bit shifting/masking in the messenger tests anymore.